app/models/manageiq/providers/nsxt/inventory/parser/network_manager.rb

```diff
@@ -146,24 +192,17 @@ def security_policy_rules(security_policy)
       security_policy_rule.action = rule['action']
       security_policy_rule.direction = rule['direction']
       security_policy_rule.ip_protocol = rule['ip_protocol']
-      security_policy_rule.source_security_groups = [] if security_policy_rule.source_security_groups.nil?
-      rule['source_groups'].each do |group|
-        next if group == 'ANY'
-        security_group = persister.security_groups.lazy_find(group.split('/groups/').last)
-        security_policy_rule.source_security_groups << security_group unless security_group.nil?
-      end
-      security_policy_rule.destination_security_groups = [] if security_policy_rule.destination_security_groups.nil?
-      rule['destination_groups'].each do |group|
-        next if group == 'ANY'
-        security_group = persister.security_groups.lazy_find(group.split('/groups/').last)
-        security_policy_rule.destination_security_groups << security_group unless security_group.nil?
-      end
-      security_policy_rule.network_services = [] if security_policy_rule.network_services.nil?
-      rule['services'].each do |service|
-        next if service == 'ANY'
-        network_service = persister.network_services.lazy_find(service.split('/services/').last)
-        security_policy_rule.network_services << network_service unless network_service.nil?
-      end
+      security_policy_rule.sources_excluded = rule['sources_excluded'].present? ? rule['sources_excluded'] : false
+      security_policy_rule.destinations_excluded = rule['destinations_excluded'].present? ? rule['destinations_excluded'] : false
+      security_policy_rule.source_security_groups = rule['source_groups']
+        .map{|group| persister.security_groups.lazy_find(group.split('/groups/').last)}
```

Comment:
Minor but prefer to have spaces here like: `.map { |group|` I'm sure miq-bot/rubocop will say the same eventually